### ISSUE TITLE
Runtime client issue when trying to run the server

### ISSUE THREAD

**Issue by PatriciaHudakova** [tatethurston/TwirpScript#48]
Hey,

I was able to follow your README example but encountered `readonly headers: Pick<Headers, "get">;` when trying to run `yarn tsc`. I have followed the readme instructions and generated files based on the same .proto

Full Logs:

```
❯ yarn tsc
yarn run v1.22.11
$ /Users/patriciahudakova/Projects/twirpscript_demo/node_modules/.bin/tsc
node_modules/twirpscript/dist/runtime/client/index.d.ts:73:28 - error TS2304: Cannot find name 'Headers'.

73     readonly headers: Pick<Headers, "get">;
                              ~~~~~~~

Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

tsconfig:

```
{
    "compilerOptions": {
      "outDir": "./build/",
      "sourceRoot": "./src/",
      "rootDir": "./src/",
      "sourceMap": true,
      "inlineSources": true,
      "module": "commonjs",
      "moduleResolution": "node",
      "lib": [
        "es2019"
      ],
      "types": [
        "node"
      ],
      "target": "es2019",
      "experimentalDecorators": true,
      "esModuleInterop": true,
      "noUnusedParameters": true,
      "noImplicitReturns": true,
      "noFallthroughCasesInSwitch": true,
      "noUncheckedIndexedAccess": true,
      "noUnusedLocals": true,
      "forceConsistentCasingInFileNames": true
    },
    "include": [
      "./src/**/*.*"
    ],
    "exclude": [
      "node_modules"
    ],
    "strict": true,
    "compileOnSave": false
}
```

**Comment by arranf** [comment]
The `Headers` interface that's being used on L97 of `src/runtime/client/index.ts` seems to refer to the `Headers` interface in `typescript/lib/lib.dom.d` (copied out and pasted below).

It looks like it needs to be imported or redefined within TwirpScript.  

```
interface Headers {
    append(name: string, value: string): void;
    delete(name: string): void;
    get(name: string): string | null;
    has(name: string): boolean;
    set(name: string, value: string): void;
    forEach(callbackfn: (value: string, key: string, parent: Headers) => void, thisArg?: any): void;
}
```

**Comment by tatethurston** [comment]
Hey @PatriciaHudakova and @arranf, thanks for reporting this and looking into the issue. @arranf is correct -- a recent change to support nodejs clients without relying on fetch made lib.dom.d.ts an implicit dependency. I've published v0.0.32 with a fix. 

Let me know if you run into any other problems, rough areas, or have any suggestions!
